app/(auth)/signin/page.tsx

```diff
@@ -57,32 +66,109 @@ export default function SignIn() {
-        <Flex flexDirection={'column'}>
-          <FormControl isInvalid={!!errors.email}>
-            <FormLabel>Email address</FormLabel>
-            <Input {...register('email')} />
-            {errors.email && (
-              <FormErrorMessage>{errors.email.message}</FormErrorMessage>
-            )}
-          </FormControl>
-          <FormControl isInvalid={!!errors.password}>
-            <FormLabel>Password</FormLabel>
-            <Input type="password" {...register('password')} />
-            {errors.password && (
-              <FormErrorMessage>{errors.password.message}</FormErrorMessage>
-            )}
-          </FormControl>
-          <PrimaryButton isLoading={isLoading} type={'submit'}>
-            Sign In
-          </PrimaryButton>
-          <Heading>{"Don't"} have an account yet?</Heading>
-          <Link href="/signup/email">Sign Up</Link>
-        </Flex>
-      </Box>
-    </>
+    <Box
+      as="main"
+      h="100vh"
+      w="100vw"
+      bg={{
+        base: 'white',
+        md: 'linear-gradient(242.77deg, #C4E6E6 1.15%, #085783 100%)'
+      }}
+      color="black"
+    >
+      <AbsoluteCenter>
+        <Card
+          bg="#F8F8F8"
+          boxShadow={{ base: '0', md: '0 4px 16px rgba(0, 0, 0, 0.1)' }}
+          maxW={{ base: '100%', lg: 'container.sm' }}
+          py={{ base: '40px', md: '40px' }}
+          px={{ base: '16px', md: '80px' }}
+        >
+          <CardHeader w="100%" p="0">
+            <Center w="100%">
+              <Image src={logo} width={144} height={40} alt="Tabi Memo" />
+            </Center>
+            <VStack
+              w="100%"
+              mt={{ base: '28px', md: '40px' }}
+              align={{ base: 'center', md: 'start' }}
+            >
+              <Heading
+                as="h1"
+                fontSize={{ base: '2xl', md: '4xl' }}
+                color="gray.700"
+              >
+                Welcome Back!
+              </Heading>
+              <Heading
+                as="h2"
+                fontSize="sm"
+                color="gray.700"
+                fontWeight="regular"
+              >
+                Sign in to your account
+              </Heading>
+            </VStack>
+          </CardHeader>
+          <CardBody
+            as="form"
+            onSubmit={signInHandler}
+            mt={{ base: '38px', md: '42px' }}
+            gap={{ base: '30px', md: '40px' }}
+            p="0%"
+            w={{ base: '80vw', md: 'unset' }}
+            maxW="500px"
+          >
+            <VStack gap="24px">
+              <FormControl isInvalid={!!errors.email}>
+                <FormLabel>Email</FormLabel>
+                <InputForm
+                  {...register('email')}
+                  minW={{ base: '300px', md: '380px' }}
+                />
+                {errors.email && (
+                  <FormErrorMessage>{errors.email.message}</FormErrorMessage>
+                )}
+              </FormControl>
```

Comment:
If you use darkmode in this app, you see like this. Could you fix this? This auth page does not care dark mode or light mode and same style!
<img width="640" alt="Screenshot 2024-02-04 at 11 15 00 AM" src="https://github.com/tabi-memo/frontend/assets/66394413/478511f2-0a89-4547-b0de-514c0c2370c8">

Reply:
Thank you for catching that! 
I fixed it and now it looks the same whether you selected dark mode or light mode :) 
All I had to do was to wrap the component in <LightMode> Tag! I had to override background and text color for autofill but it looks good now. 
```typescript
 const signInInputStyle = {
    'input:-webkit-autofill, input:-webkit-autofill:focus': {
      boxShadow: '0 0 0 1000px #E8F0FE inset',
      textFillColor: 'black'
    }
  }

``` 

<img src="https://github.com/tabi-memo/frontend/assets/70562492/d92d857e-73ad-46cd-b2a2-451a2a5826e7" width="300" />